packages/examples/tsconfig.json

```diff
@@ -1,6 +1,14 @@
 {
   "extends": "../../tsconfig.packages.json",
   "compilerOptions": {
-    "baseUrl": "./"
+    "baseUrl": "./",
+    "composite": false,
+    "module": "CommonJS",
+    "moduleResolution": "Node10",
+    "paths": {
+      "@metamask/snaps-sdk/jsx-dev-runtime": ["../../../snaps-sdk/src/jsx/jsx-dev-runtime.ts", "../snaps-sdk/src/jsx/jsx-dev-runtime.ts"],
+      "@metamask/snaps-sdk/jsx": ["../../../snaps-sdk/src/jsx", "../snaps-sdk/src/jsx"],
+      "@metamask/snaps-sdk": ["../../../snaps-sdk/src", "../snaps-sdk/src"],
+    }
```

Comment:
These changes make the `// eslint-disable-next-line @typescript-eslint/no-throw-literal` comments in the examples unnecessary. Before, it would depend on build output whether these comments were necessary or not.